tools/tsp-client/package.json

```diff
@@ -38,10 +38,12 @@
   ],
   "devDependencies": {
     "@types/chai": "^4.3.5",
+    "@types/lodash": "^4.17.16",
     "@types/node": "^20.4.8",
     "@types/prompt-sync": "^4.2.1",
     "@typespec/compiler": "^1.0.0-0",
     "chai": "^4.3.7",
+    "lodash": "^4.17.21",
```

Comment:
i would try not to use lodash, its now an old package that hasn't been updated in 4 years